src/current/_includes/releases/v23.2/v23.2.0-alpha.3.md

```diff
@@ -105,8 +105,8 @@ Release Date: October 10, 2023
     |---------------------------------------|----------------------------------------------------|
     | `server.shutdown.drain_wait`          | `server.shutdown.initial_wait`                     |
     | `server.shutdown.lease_transfer_wait` | `server.shutdown.lease_transfer_iteration.timeout` |
-    | `server.shutdown.query_wait`          | `server.shutdown.queries.timeout`                  |
-    | `server.shutdown.connection_wait`     | `server.shutdown.transactions.timeout`             |
+    | `server.shutdown.query_wait`          | `server.shutdown.transactions.timeout`             |
```

Comment:
I do not understand this. Were the previous release notes incorrect?  

If we go back and say this about the v23.2.0-alpha.3 either the docs were wrong (totally possible) or ... ?

Reply:
The release note was incorrect. This fixes it.

Reply:
Thank you for fixing it!